inc/admin/namespace.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * @package Aldine
+ */
+
+namespace Aldine\Admin;
+
+use PressbooksMix\Assets;
+
+const BLOG_OPTION = 'pressbooks_aldine_in_catalog';
```

Comment:
How do we handle the old option from networks that have been using Pressbooks Publisher?